lib/cn/cn_tree_internal.h

```diff
@@ -110,6 +112,7 @@ struct cn_tree {
     struct kvs_cparams *ct_cp;
     u64                 cnid;
     u64                 ct_dgen_init;
+    uint                ct_splitgen;
```

Comment:
`uint` should be banned since it is a deprecated typedef!

Reply:
Would you be ok with u_int, u_long, ... from __USE_MISC.   The excessive verbosity of "unsigned xxx" serves no useful purpose...

Reply:
I just think we should use what Ritchie gave us :). I don't disagree that typing `unsigned` blows chunks, but what does blow even more chunks is having to include an extra header for a problem that can be solved by the keywords the language already gives us. `u_int` and friends don't seem to be guaranteed to exist given they aren't mentioned in `sys_types.h(0P)`.

Reply:
I would love to adopt a shortened version of unsigned scalar types.

Reply:
u_int and friends are from __USE_MISC (see features.h).  uint and u_int go waaaay back, they must continue to exist in order to be able to compile 30+ year old C code :-)

Reply:
My understanding is that Ritchie thought enums were kinda stupid (or maybe stupidly implemented?).  He was very polite about it though, as one would expect, calling them "curious" :-)
You missed the good times before prototypes, and then the extra good times where you had to manage your prototypes by hand.  At any rate, I removed this field from the patch ;-)

Reply:
I only hope that people debate my stupid choices 50 years from now too.

Reply:
Heh, I don't think that will be a problem with the rate people these days toss out everything "old" and reinvent it all from scratch!